src/sentry/integrations/slack/client.py

```diff
@@ -34,17 +34,17 @@ def track_response_data(
         # If Slack gives us back a 200 we still want to check the 'ok' param
         if resp:
             content_type = resp.headers["content-type"]
-            if content_type == "application/json":
```

Comment:
@mgaeta I think we shouldn't assume that if the content type is `application/json` if it isn't `text/html`. I'd be really surprised if Slack isn't sending us the right header for JSON all of the sudden, especially if we aren't hearing reports of things breaking. Can we just add some logging or metrics for the content types we are getting?

Reply:
We're pretty sure that my PR caused the graph to invert. Previous to that PR we didn't check the content-type at all and just called ```resp.json().get("ok")```. We added a condition for HTML because Slack responds with `b"ok"` when you send and ephemeral message.

Reply:
@mgaeta can you link to that PR?

Reply:
nvm, i see that's already there